pkg/virt-config/config_test.go

```diff
@@ -141,14 +121,17 @@ var _ = Describe("ConfigMap", func() {
 		table.Entry("is invalid, GetNodeSelectors should return the default", "-1", nil),
 	)
 
-	table.DescribeTable(" when machineType", func(value string, result string) {
+	table.DescribeTable(" when machineType", func(cpuArch string, machineType string, result string) {
 		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
-			Data: map[string]string{virtconfig.MachineTypeKey: value},
+			Data: map[string]string{virtconfig.MachineTypeKey: machineType},
 		})
+		clusterConfig.SetCPUArch(cpuArch)
```

Comment:
Maybe we can move this `SetCPUArch` call into `NewFakeClusterConfig` and set it on the construction of the config? As consequence we could get rid of `SetCPUArch` which is only used for testing.

Reply:
OK

Reply:
Updated.